pkg/plugin/plugin_manager.go

```diff
@@ -51,7 +59,10 @@ func LoadPlugins(dir string) (map[string]jvspb.Validator, *multicloser.Closer, e
 
 		// Enable the plugin.
 		pluginClient := plugin.NewClient(&plugin.ClientConfig{
-			HandshakeConfig:  jvspb.Handshake,
+			HandshakeConfig: jvspb.Handshake,
+			// Plugins field need to be set otherwise exception will be thrown from
+			// https://github.com/hashicorp/go-plugin/blob/a88a423a8813d0b26c8e3219f71b0f30447b5d2e/client.go#L909
+			Plugins:          pluginMap,
```

Comment:
Why the `pluginMap` is static? Shouldn't it be based on the `name`? E.g.
```suggestion
			Plugins:          map[string]plugin.Plugin{name: &jvspb.ValidatorPlugin{}},
```

Reply:
My understanding is pluginMap demonstrates host's knowledge about each plugin as some validation check per the code pointer [here](https://github.com/hashicorp/go-plugin/blob/a88a423a8813d0b26c8e3219f71b0f30447b5d2e/examples/grpc/shared/interface.go#L26). If we make it a dynamic variable, we pretty much allows all the plugins in the dir. I have adjusted the code as suggest for now since it avoids the need to update the pluginMap every time a new plugin is introduced. @sethvargo let us know whether we use it the correct way.

Reply:
Resolving now to unblock the merge. @sethvargo feel free to keep commenting in this thread if any change is needed, I will send out follow up PR to make the change.